src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -262,15 +267,19 @@ function WorkspaceInitialPage(props) {
                                         )}
                                     </View>
                                 </View>
+                                {/*
+                                    Ideally we should use MenuList component for MenuItems with singleExecution/Navigation Actions.
+                                    But Here we need to have a `isExecuting` for profile details click actions also so we are directly mapping menuItems.
+                                */}
                                 {_.map(menuItems, (item) => (
                                     <MenuItem
                                         key={item.translationKey}
-                                        disabled={hasPolicyCreationError}
+                                        disabled={hasPolicyCreationError || isExecuting || isCurrencyModalOpen}
```

Comment:
Why we're adding isCurrencyModalOpen?

Reply:
I see, thanks for clarifying. I think we should have a more general solution here - 
We can do what you implemented, but in a generalized way. Like disable list when modal is open in foreground. But adding this condition to every modal on the page does not sound like a good option as it is easy to miss and understand the context, and will eventually lead to blink bug every time a modal is added. Thoughts?

Reply:
The `modal` onyx key! cc @robertKozik